### DISCUSSION TITLE
Is possible drag drop open an custom menu for commads ?

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1367] [General]
Hello.
Thanks alexx2000 and users for reading my topic.
Double Commander allow when selecting an file or folder for drag drop open an menu with sub menus for custom commands ?
Select an file for open menu "PEAZIP" and an sub menu with custom commands ( extract files, extract here, compress in 7Z maximum, compress minimum compression and etc).
The user create the commands inside menus.
That save much time when managing files and help Linux users.
I not known as to do it in Linux and unhappily Double Commander not allow file operation in tree view panel too.

Have an nice week.

**Comment by alexx2000** [comment]
Hi,
What menu do you mean? Is it a menu opened after drag&drop or menu opened by right mouse click?

**Comment by Augusto7743** [comment]
Thanks very much for replying.
Both drag & drop and right button.
I not known if is possible to do it in Linux.
In windows is much simple and fast manage file in explorer with 7Z installed.
Linux is more hard not simple for some users.

**Comment by alexx2000** [comment]
Drag&drop - no. Right mouse button menu - yes, but without submenus. You can add user commands into context menu using internal [file associations](https://doublecmd.github.io/doc/en/configuration.html#ConfigAssociations).

**Comment by Augusto7743** [comment]
Thanks very much for replying.

I will try configure some commands and if Peazip allow to be used as command line.
In Linux need start Peazip ... after select the path or some files ... after the path to save and close the software.
In other OS is simple create 7Z files using drag & drop between folders paths.

I use Double Commander with tree view enabled. Help very much browsing paths, but need press enter for each path and not allow file operations between the tree view tab and window tab.
When browsing in tree view If need view several paths need press enter several times for show files in window tab.

![1](https://github.com/doublecmd/doublecmd/assets/21184987/9d111b26-d98f-4f75-a772-16d88c3a840a)


When I need use 2 windows tabs only need hold the mouse pointer in line in right side and move the line to left.



![2](https://github.com/doublecmd/doublecmd/assets/21184987/850c3968-a0ab-49d6-b7e3-bfe0b68b3033)





Your software is gold for Linux advanced users.
Have an nice week.
